authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -429,25 +432,30 @@ func getAuthInfoListForWildcard(subRevReq *authzv1.SubjectAccessReviewSpec, stor
 func getAuthInfoListForCustomResource(subRevReq *authzv1.SubjectAccessReviewSpec, clusterType string) ([]azureutils.AuthorizationActionInfo, error) {
 	var authInfoList []azureutils.AuthorizationActionInfo
 	if subRevReq.ResourceAttributes.Verb == "*" {
+		klog.V(7).InfoS("CustomResourceTypeCheck: Creating actions for all custom resource verbs", "group", subRevReq.ResourceAttributes.Group, "resource", subRevReq.ResourceAttributes.Resource)
 		for _, action := range getCustomResourceOperationsMap(clusterType) {
 			authInfoList = append(authInfoList, action)
 		}
 	} else {
 		action := getActionName(subRevReq.ResourceAttributes.Verb)
 		authInfoSingle, found := getCustomResourceOperationsMap(clusterType)[action]
 		if !found {
+			klog.ErrorS(errors.New("action not found"), "CustomResourceTypeCheck: No actions found for custom resource verb", "verb", subRevReq.ResourceAttributes.Verb, "group", subRevReq.ResourceAttributes.Group, "resource", subRevReq.ResourceAttributes.Resource)
 			return nil, errors.Errorf("No actions found for verb %s", subRevReq.ResourceAttributes.Verb)
 		}
+		klog.V(7).InfoS("CustomResourceTypeCheck: Creating action for custom resource", "group", subRevReq.ResourceAttributes.Group, "resource", subRevReq.ResourceAttributes.Resource, "verb", subRevReq.ResourceAttributes.Verb, "action", action)
 		authInfoList = append(authInfoList, authInfoSingle)
 	}
 
 	for i := range authInfoList {
 		err := setAuthInfoResourceAttributes(&authInfoList[i], subRevReq)
 		if err != nil {
+			klog.ErrorS(err, "CustomResourceTypeCheck: Error while setting resource attributes", "group", subRevReq.ResourceAttributes.Group, "resource", subRevReq.ResourceAttributes.Resource)
 			return nil, errors.Errorf("Error while setting resource attributes: %s", err.Error())
```

Comment:
where is this error observed/logged? if it's already logged, i suggest removing the log above.